framework/packages/abstract-client/src/account.rs

```diff
@@ -160,18 +160,84 @@ impl<Chain: CwEnv> Account<Chain> {
     // Install an application on the account
     // creates a new sub-account and installs the application on it.
     pub fn install_app<
-        M: ContractInstance<Chain> + RegisteredModule + From<Contract<Chain>> + Clone,
+        M: ContractInstance<Chain> + InstallConfig + From<Contract<Chain>> + Clone,
     >(
         &self,
         configuration: &M::InitMsg,
         funds: &[Coin],
+    ) -> AbstractClientResult<Application<Chain, M>> {
+        self.install_app_internal(vec![M::install_config(configuration)?], funds)
+    }
+
+    pub fn install_app_with_dependencies<
+        M: ContractInstance<Chain>
+            + DependencyCreation
+            + InstallConfig
+            + From<Contract<Chain>>
+            + Clone,
+    >(
+        &self,
+        module_configuration: &M::InitMsg,
+        dependencies_config: M::DependenciesConfig,
+        funds: &[Coin],
+    ) -> AbstractClientResult<Application<Chain, M>> {
+        let mut install_configs: Vec<ModuleInstallConfig> =
+            M::dependency_install_configs(dependencies_config)?;
+        install_configs.push(M::install_config(module_configuration)?);
+
+        self.install_app_internal(install_configs, funds)
+    }
+
+    pub fn direct_admin(&self) -> AbstractClientResult<Addr> {
+        self.abstr_account.manager.address().map_err(Into::into)
+    }
+
+    pub fn account_owner(&self) -> AbstractClientResult<cw_ownable::Ownership<String>> {
```

Comment:
We return `String` and not `Addr` from the manager since we use the [cw_ownable_query](https://docs.rs/cw-ownable/latest/cw_ownable/attr.cw_ownable_query.html) macro. Weird to me that they defaulted to a `String` and not `Addr` though.

Reply:
Yeah IMO it should return Addr. However, we can change the query to let it return `Ownership<Addr>` if we want because `struct Addr(String)` is serialized/de-serialized just like a String.